src/components/DisplayedProfile/DisplayedProfile.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+// import { useNavigate } from 'react-router-dom';
+import { Flex, FormLabel, Input, Image } from '@chakra-ui/react';
+import profilePicture from '../../assets/profilePicture.png';
```

Comment:
🚫 **[eslint]** <[import/no-unresolved](https://github.com/import-js/eslint-plugin-import/blob/v2.27.5/docs/rules/no-unresolved.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unable to resolve path to module '../../assets/profilePicture.png'.